src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -65,18 +68,20 @@
 	private Path clientJarSrg;
 	private IMappingProvider primaryMappings;
 	private List<IMappingProvider> devMappings;
-	private FieldDescriptorProvider fieldDescriptorProvider;
-	private NaiveRemapper naiveRemapper;
-	private AccessTransformerRemapper accessTransformerRemapper;
+	private PatchworkRemapper patchworkRemapper;
+	private Remapper accessTransformerRemapper;
+	private final IntermediaryHolder intermediaryHolder;
 	private boolean closed = false;
 
-	public Patchwork(Path inputDir, Path outputDir, Path dataDir, Path tempDir, IMappingProvider primaryMappings, List<IMappingProvider> devMappings) {
+	public Patchwork(Path inputDir, Path outputDir, Path dataDir, Path tempDir, Path bridgedMappings, List<IMappingProvider> devMappings) {
 		this.inputDir = inputDir;
 		this.outputDir = outputDir;
 		this.dataDir = dataDir;
 		this.tempDir = tempDir;
 		this.clientJarSrg = dataDir.resolve(version + "-client+srg.jar");
-		this.primaryMappings = primaryMappings;
+		this.primaryMappings = TinyUtils.createTinyMappingProvider(bridgedMappings, "srg", "intermediary");
+		this.intermediaryHolder = new IntermediaryHolder(TinyUtils.createTinyMappingProvider(bridgedMappings, "intermediary", "srg"));
```

Comment:
Seems like this can just as easily be done with the base intermediary mappings instead of having to use bridged mappings?